### PR TITLE
sync

### PR DESCRIPTION
**Pull request by Mandarancio**

### CODE REVIEW

File: plugins/language_marte.lua [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+      pattern = "Type%s+()=()%s+[%a_][%w_]*",
+      type = { "keyword", "operator", "keyword2"}
+    },
+    {
+      pattern = "[%+%$][%a_][%w_]+%s()=",
+      type = {"function", "operator"}
+    },
+    { pattern = "=%s+()[%a_][%w_]+", type = "string" },
```
Comment (Guldoman) [inline plugins/language_marte.lua:28]: This is dividing the match in two `()`, but only a single token type is provided.

File: README.md
Comment (Guldoman) [inline README.md:1]: No need to update the README, it'll be automatically updated from the manifest info.

**Review by Guldoman — COMMENTED** [review]
Could you also add the new languages as dependencies of `meta_languages`?
Bump its version too.

**Comment by Guldoman** [comment]
Wait, I'm a bit confused about what's going on here...

Still, the comment about the MARTe language stands.

**Comment by Mandarancio** [comment]
Sorry my bad, I did not want to publish this pull request, I will delete it...

**Pull request closed without merging**